extensions/query-store/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-Query Store extension for Azure Data Studio simplifies assists users with troubleshooting query performance in SQL Server and Azure SQL and understanding issues caused by plan changes.
+The Query Store extension for Azure Data Studio simplifies and assists users with troubleshooting query performance in SQL Server and SQL Database, and with understanding issues caused by plan changes.
```

Comment:
nit: since you're refining the wording, you may say "execution plan" instead of "plan"